.travis.yml

```diff
@@ -2,19 +2,24 @@ language: python
 install:
   - pip install tox
 dist: xenial
-python: 3.5
+python: 3.6  # 3.6 is available in the xenial image by default.
 script:
     - tox -e $TOX_ENV
 matrix:
   include:
     # TEST DOCS AND LINTING
+    # Use default python3 version here.
     - env: TOX_ENV=py3-docs
     - env: TOX_ENV=py3-lint
```

Comment:
I think linting target the oldest version of python we support. There have been some changes to python's syntax in python 3.6 and python 3.7 (e.g. lazy evaluations of types in hinting). I could imagine using those features results in correct linting in the most recent versions, but may actually give errors in older versions.

Reply:
:+1: Will default back to 3.5 then.